entityx/help/Pool_test.cc

```diff
@@ -13,8 +13,9 @@
 #include <vector>
 #include "entityx/3rdparty/catch.hpp"
 #include "entityx/help/Pool.h"
+#include "entityx/Entity.h"
 
-struct Position {
+struct Position : public entityx::Component<Position> {
```

Comment:
@alecthomas It turns out that it works after all, and the error that I encountered was a fluke and unrelated. I reverted this change so the POD types work fine.